python/timeplus/dbapi.py

```diff
@@ -102,8 +111,24 @@ def __init__(
         address: String - Server address. Default is "https://us.timeplus.cloud".
         apikey: String - API key for the connection. Default is None.
         workspace: String - Workspace for the connection. Default is None.
+        username: String - username for the connection. Default is None.
+        password: String - password for the connection. Default is None.
         """
-        self.env = Environment().address(address).workspace(workspace).apikey(apikey)
+
+        if os.environ.get("TIMEPLUS_SQLALCHEMY_SCHEMA") == "http":
+            address = address.replace("https://", "http://")
+
+        self.env = Environment().address(address).workspace(workspace)
+
+        if apikey is not None:
+            self.env.apikey(apikey)
+
+        if username is not None:
+            self.env.username(username)
+
+        if password is not None:
```

Comment:
it can be none

Reply:
if the password is None, dont set it is OK?